codegen/projections/white_label/sig/white_label/types.rbs

```diff
@@ -126,6 +126,14 @@ module WhiteLabel
       include Hearth::Structure
     end
 
+    class RelativeOperationInput < ::Struct[untyped]
+      include Hearth::Structure
```

Comment:
I think you may have to merge main and rebuild.

Reply:
!!! yes, thanks for reminder :)